packit_service/worker/helpers/job_helper.py

```diff
@@ -167,7 +167,9 @@ def _report(
             description = (
                 f"{description}\n\n---\n*Packit-User does not have access to the "
                 "source project. We have requested access to be able to set "
-                "commit statuses / pipelines instead of the comments.*"
+                "commit statuses / pipelines instead of the comments."
+                "For merge-request the user needs to be added to the forked "
+                "project that is the source for the merge-request.*"
```

Comment:
If I understand it correctly, your added sentence explains the first sentence. So I'd perhaps merge them. Or maybe just replace the first one with yours (and s/user/Packit-User/)?

Reply:
I think it just extends it. Explaining more in deep what source project means in case of merge-request. Do you have a better idea for the whole paragraph?

Reply:
What about this?

```python
              f"{description}\n\n---\n*Packit-User does not have access to the "
                "source project (=usually author's fork of the project). "
                "We have requested access to be able to set "
                "commit statuses / pipelines instead of the comments. "
                "(This is only about the representation of the results. "
                "Packit is still able to do its job without having the permissions.)*"
```

Reply:
Not sure if we should mention that it's because of how GitLab works...

Reply:
I like Franta's version.

Reply:
I used Franta's version.